howto/change-python.md

```diff
@@ -0,0 +1,32 @@
+# Bump Python Version for DjangoGirls.org Website
+
+In order to bump up the Python version for the DjangoGirls.org website, you 
+need to have access to our PythonAnywhere account, and follow these instructions:
+
+1. [Login to PythonAnywhere](https://www.pythonanywhere.com/user/djangogirls2/consoles/). Credentials can be found in 1Password.
+2. Check if the Python version you want to upgrade to is supported by the current
+system image used on PythonAnywhere by clicking on the pen icon on the Web Tab or
+by [reading this](https://help.pythonanywhere.com/pages/ChangingSystemImage#:~:text=Once%20you%20have%20changed%20the,again%20to%20pick%20it%20up.).
+ If yes, no need to change the system image. If system image needs to be changed, follow the instructions here.
+3. Check which node version we are using opening the `Bash Console` and type 
+```which node```
```

Comment:
No it wouldn't. By the time you get to need this information, you will not be able to get it because the image would already have been changed and you won't be able to get it.